ext/django_stubs_ext/compat.py

```diff
@@ -0,0 +1,4 @@
+from typing import Literal
+
+# Magic constant that mypy overrides to True during type checking
+MYPY: Literal[False]
```

Comment:
```suggestion
MYPY: Literal[False] = False
```

I guess it should have a runtime value as well.